posts/2021-10-29-kube-secrets-210011.adoc

```diff
@@ -84,10 +84,10 @@ For more information visit link:https://kubernetes.io/docs/concepts/configuratio
 [#user_features]
 == Enable installation of user features on Open Liberty
 
-The FeatureUtility command-line tool, Liberty Maven and Gradle Plugin will now enable installation of user features onto Open Liberty runtime from Maven Central and on-premise Maven repository. Prior to this update, users had to use `InstallUtility` command-line tool from commercial liberty.
+The `featureUtility` command-line tool, Liberty Maven and Gradle plugins will now enable installation of user features onto Open Liberty runtime from Maven Central and on-premises Maven repository.
```

Comment:
suggested edit:

You can now install user features onto the Open Liberty runtime from Maven Central or on-premises Maven repository by using the `featureUtility` command-line tool and the Liberty Maven and Gradle plug-ins.